docs-site/docs/docs/about-menu.md

```diff
@@ -0,0 +1,43 @@
+The about menu provides a deeper view into your instance of Polynote. 
+
+![Settings Menu](images/settings-menu.png)
+
+The left pane contains several sections to view different pieces of data about Polynote. 
+
+### About 
+The about menu contains information about your instance's version and most recent server commit. 
+
+### Hotkeys 
+The hotkeys menu contains a list of all supported keyboard shortcuts in Polynote. 
+
+### Preferences 
+The preferences menu contains two sections - _UI preferences_ and _storage_.
+
+#### UI Preferences 
+The preferences menu is where you can customize the following settings: 
+
+- **vim**: Whether VIM input mode is enabled for code cells 
+- **theme**: Light or Dark application color scheme 
+- **notifications**: Whether Polynote can send you browser notifications. 
+
+#### Storage 
```

Comment:
Here's where a nice minimal example of a few stored things and an arrow pointing out that trash can would be neat. I'm sure we can display this in a much nicer way but for now that trash can is a little hard to see.